colossalai/nn/parallel/data_parallel.py

```diff
@@ -191,8 +193,11 @@ class ZeroDDP(ColoDDP):
             For more details, see the API reference of ``GeminiManager``.
     """
 
-    def __init__(self, module: torch.nn.Module, gemini_manager: GeminiManager) -> None:
-        super().__init__(module.half())
+    def __init__(self,
+                 module: torch.nn.Module,
+                 gemini_manager: GeminiManager,
+                 process_group: Optional[ColoProcessGroup] = None) -> None:
```

Comment:
ColoDDP receives `dist.ProcessGroup`, and ZeroDDP receives `ColoProcessGroup`. Is this difference necessary?

Reply:
I will update it in the next PR.

Reply:
updated, both of them use ColoProcessGroup